scripts/ping-providers.ts

```diff
@@ -50,9 +46,8 @@ async function validateLatestBlock(client: PublicClient, chain: Chain): Promise<
   const block = blockRes.data;
   const blockTimestamp = block.timestamp;
   const deltaTime = Number(BigInt(Math.floor(new Date().getTime() / 1000)) - blockTimestamp);
-  // The block time in seconds multiplied by an arbitrary tolerance multiplier
-  const tolerance = (chain.blockTimeMs / 1_000) * BLOCK_TOLERANCE_MULTIPLIER;
-  if (Math.abs(deltaTime) > tolerance) {
+  const cutoff = 60 * 5; // Arbitrary cutoff of 5 minutes
+  if (Math.abs(deltaTime) > cutoff) {
```

Comment:
I chose 5 min because it's very conservative for most chains, but perhaps there is a better way to do this?

Reply:
That's fine except for extremely inactive roll ups. For those, the only way to check if the chain/provider isn't stuck is to send a transaction and see it getting confirmed, which we wouldn't bother to do.